### PR TITLE
Edited README.txt for better readability

### PR DESCRIPTION
**Pull request by Philip-Choi**
1. Deleted Article 'the' and empty words
2. Edited repetition
3. Deleted empty phrase 'There is'  
4. Deleted pronouns and added hyperlinks
5. Deleted empty phrase 'There is' and changed the sentence to active form

Thank you!

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -19,16 +19,16 @@ Some libraries Autopsy uses may have different, but similar, open source license
 INSTALLATION
 
 For a Windows installation, all Autopsy dependencies are bundled with the installer provided.
-There is no need for manual installation of additional dependencies if the Windows installer is used.
+Manual installation of additional dependencies are not required if the Windows installer is used.
 
-If you want the Japanese localized version, you must have the Japanese language pack (http://support.microsoft.com/kb/972813) installed and the default locale set to JA. (http://windows.microsoft.com/en-us/windows/change-system-locale#1TC=windows-7).
+If the Japanese localized version is required, the [Japanese language pack](http://support.microsoft.com/kb/972813) must be installed and the default locale must be [set to JA](http://windows.microsoft.com/en-us/windows/change-system-locale#1TC=windows-7).
```
Comment (APriestman) [inline README.txt]: This is just a text file designed to be open in notepad or some other simple reader. The brackets won't make a link.

File: README.txt [hunk 2]
```diff
@@ -19,16 +19,16 @@ Some libraries Autopsy uses may have different, but similar, open source license
...
 
 Refer to the KNOWN_ISSUES.txt file for known bugs that could cause investigation problems. 
 
 
 SUPPORT
 
-There is a built-in help system in Autopsy once you get it started.  There is also a QuickStart Guide that comes with the installer.
+Built-in help system exist once Autopsy starts and an QuickStart Quide comes with the installer.
```
Comment (APriestman) [inline README.txt]: Put an 'A' at the beginning
exist->exists
an QuickStart -> a QuickStart
Quide -> guide

**Comment by rcordovano** [comment]
Hi, @Philip-Choi, thanks for your contribution so far. Are you planning to respond to the reviewer's comments and make additional commits, or should this pull request be closed?

**Comment by Philip-Choi** [comment]
You can close it. thank you

________________________________

**Pull request closed without merging**